lib/systemd.c

```diff
@@ -86,6 +86,7 @@ static int systemd_get_watchdog_time(int the_process)
 
 void systemd_send_stopping(void)
 {
+	systemd_send_information("STATUS=");
```

Comment:
status == FRR operational indicate that all the daemons are successfully started and running. Also, if a particular daemon is down or continuously restarting, the status would be "restarting". Right ?

if two daemons are restarting at any given instance, does this show two daemons with status as restarting ?

Reply:
say bgpd died you would see `status==restarting bgpd` and on zebra death you would see `status == restarting all`